packages/sdk-components-react/src/xml-time.ws.ts

```diff
@@ -0,0 +1,22 @@
+import { CalendarIcon } from "@webstudio-is/icons/svg";
+
+import {
+  type WsComponentMeta,
+  type WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+
+import { props } from "./__generated__/xml-time.props";
+
+export const meta: WsComponentMeta = {
+  category: "xml",
+  type: "container",
+  description: "Converts machine-readable date and time to ISO format.",
```

Comment:
machine-readable? or just unix time stamp? what are all the formats it supports?

Reply:
Have no idea, took same description as it was in time